website/website/add_youtuber.py

```diff
@@ -0,0 +1,77 @@
+from googleapiclient.discovery import build
+from urllib.parse import urlparse, unquote
+import os
```

Comment:
I'd consider isort https://pycqa.github.io/isort/ here to sort imports.

It's one of the best practices related to imports.